
> offcorss-assistant@1.0.0 start /home/ubuntu/offcorss-assistant
> node index.js

App listening on port 80.

> offcorss-assistant@1.0.0 start /home/ubuntu/offcorss-assistant
> node index.js

App listening on port 80.
events.js:160
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property 'password' of null
    at /home/ubuntu/offcorss-assistant/routes/users.js:117:47
    at Query.<anonymous> (/home/ubuntu/offcorss-assistant/node_modules/mongoose/lib/model.js:3755:16)
    at /home/ubuntu/offcorss-assistant/node_modules/kareem/index.js:277:21
    at /home/ubuntu/offcorss-assistant/node_modules/kareem/index.js:131:16
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)

npm ERR! Linux 4.4.0-1013-aws
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! offcorss-assistant@1.0.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the offcorss-assistant@1.0.0 start script 'node index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the offcorss-assistant package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs offcorss-assistant
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls offcorss-assistant
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/offcorss-assistant/npm-debug.log

> offcorss-assistant@1.0.0 start /home/ubuntu/offcorss-assistant
> node index.js

App listening on port 80.
events.js:160
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property 'password' of null
    at /home/ubuntu/offcorss-assistant/routes/users.js:117:47
    at Query.<anonymous> (/home/ubuntu/offcorss-assistant/node_modules/mongoose/lib/model.js:3755:16)
    at /home/ubuntu/offcorss-assistant/node_modules/kareem/index.js:277:21
    at /home/ubuntu/offcorss-assistant/node_modules/kareem/index.js:131:16
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)

npm ERR! Linux 4.4.0-1013-aws
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! offcorss-assistant@1.0.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the offcorss-assistant@1.0.0 start script 'node index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the offcorss-assistant package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs offcorss-assistant
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls offcorss-assistant
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/offcorss-assistant/npm-debug.log

> offcorss-assistant@1.0.0 start C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back
> node index.js

App listening on port 80.
connection error: { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\topologies\server.js:328:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\pool.js:280:12)
    at Connection.g (events.js:292:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\connection.js:177:49)
    at Socket.g (events.js:292:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1277:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
  name: 'MongoError',
  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }
(node:17300) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
connection error: { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\topologies\server.js:328:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\pool.js:280:12)
    at Connection.g (events.js:292:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\connection.js:177:49)
    at Socket.g (events.js:292:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1277:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
  name: 'MongoError',
  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }
connection error: { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\topologies\server.js:328:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\pool.js:280:12)
    at Connection.g (events.js:292:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\connection.js:177:49)
    at Socket.g (events.js:292:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1277:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
  name: 'MongoError',
  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }
(node:17300) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
connection error: { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\topologies\server.js:328:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\pool.js:280:12)
    at Connection.g (events.js:292:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\connection.js:177:49)
    at Socket.g (events.js:292:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1277:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
  name: 'MongoError',
  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }
connection error: { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\topologies\server.js:328:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\pool.js:280:12)
    at Connection.g (events.js:292:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\connection.js:177:49)
    at Socket.g (events.js:292:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1277:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
  name: 'MongoError',
  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }
connection error: { MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\topologies\server.js:328:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\pool.js:280:12)
    at Connection.g (events.js:292:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\node_modules\mongoose\node_modules\mongodb-core\lib\connection\connection.js:177:49)
    at Socket.g (events.js:292:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1277:8)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
  name: 'MongoError',
  message: 'failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]' }
(node:17300) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]

npm ERR! Windows_NT 10.0.16199
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! offcorss-assistant@1.0.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the offcorss-assistant@1.0.0 start script 'node index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the offcorss-assistant package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs offcorss-assistant
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls offcorss-assistant
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\aleja\OneDrive\Documentos\BASE16\OFFCORSS\Asistente\OffcorssAssistant-back\npm-debug.log

> offcorss-assistant@1.0.0 start /home/ubuntu/offcorss-assistant
> node index.js

App listening on port 80.

> offcorss-assistant@1.0.0 start /home/ubuntu/offcorss-assistant
> node index.js

events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE :::80
    at Object.exports._errnoException (util.js:1018:11)
    at exports._exceptionWithHostPort (util.js:1041:20)
    at Server._listen2 (net.js:1258:14)
    at listen (net.js:1294:10)
    at Server.listen (net.js:1390:5)
    at EventEmitter.listen (/home/ubuntu/offcorss-assistant/node_modules/express/lib/application.js:618:24)
    at Object.<anonymous> (/home/ubuntu/offcorss-assistant/index.js:22:5)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3

npm ERR! Linux 4.4.0-1013-aws
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! offcorss-assistant@1.0.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the offcorss-assistant@1.0.0 start script 'node index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the offcorss-assistant package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs offcorss-assistant
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls offcorss-assistant
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/offcorss-assistant/npm-debug.log

> offcorss-assistant@1.0.0 start /home/ubuntu/offcorss-assistant
> node index.js

App listening on port 80.
(node:17742) Warning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
Terminated

npm ERR! Linux 4.4.0-1013-aws
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! offcorss-assistant@1.0.0 start: `node index.js`
npm ERR! Exit status 143
npm ERR! 
npm ERR! Failed at the offcorss-assistant@1.0.0 start script 'node index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the offcorss-assistant package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs offcorss-assistant
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls offcorss-assistant
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/offcorss-assistant/npm-debug.log
2017-06-21T15:16:39.337+0000 I CONTROL  [initandlisten] MongoDB starting : pid=18754 port=27017 dbpath=/data/db 64-bit host=ip-172-31-18-184
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten] db version v3.2.14
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten] git version: 92f6668a768ebf294bd4f494c50f48459198e6a3
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten] modules: none
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten] build environment:
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten]     distarch: x86_64
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2017-06-21T15:16:39.338+0000 I CONTROL  [initandlisten] options: {}
2017-06-21T15:16:39.366+0000 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2017-06-21T15:16:39.366+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=1G,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] 
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] 
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] 
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/defrag is 'always'.
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-06-21T15:16:39.601+0000 I CONTROL  [initandlisten] 
2017-06-21T15:16:39.602+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2017-06-21T15:16:39.602+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
2017-06-21T15:16:39.602+0000 I NETWORK  [HostnameCanonicalizationWorker] Starting hostname canonicalization worker
2017-06-21T15:16:43.530+0000 I NETWORK  [initandlisten] connection accepted from 190.7.156.123:10835 #1 (1 connection now open)
2017-06-21T15:16:52.478+0000 I NETWORK  [initandlisten] connection accepted from 190.7.156.123:34704 #2 (2 connections now open)
2017-06-21T15:16:55.326+0000 I NETWORK  [initandlisten] connection accepted from 127.0.0.1:58554 #3 (3 connections now open)
2017-06-21T15:16:55.362+0000 I NETWORK  [initandlisten] connection accepted from 127.0.0.1:58556 #4 (4 connections now open)
2017-06-21T15:16:55.369+0000 I INDEX    [conn3] build index on: assistant.users properties: { v: 1, unique: true, key: { username: 1 }, name: "username_1", ns: "assistant.users", background: true }
2017-06-21T15:16:55.369+0000 I INDEX    [conn3] build index done.  scanned 1 total records. 0 secs
2017-06-21T15:16:55.377+0000 I NETWORK  [conn3] end connection 127.0.0.1:58554 (3 connections now open)
2017-06-21T15:16:55.378+0000 I NETWORK  [conn4] end connection 127.0.0.1:58556 (2 connections now open)
2017-06-21T15:17:10.812+0000 I NETWORK  [initandlisten] connection accepted from 127.0.0.1:58558 #5 (3 connections now open)
2017-06-21T15:17:10.815+0000 I NETWORK  [conn5] end connection 127.0.0.1:58558 (2 connections now open)
2017-06-21T15:17:14.044+0000 I NETWORK  [initandlisten] connection accepted from 127.0.0.1:58560 #6 (3 connections now open)
2017-06-21T15:17:14.046+0000 I NETWORK  [conn6] end connection 127.0.0.1:58560 (2 connections now open)
2017-06-21T15:17:16.620+0000 I NETWORK  [initandlisten] connection accepted from 127.0.0.1:58562 #7 (3 connections now open)
2017-06-21T15:17:16.626+0000 I NETWORK  [conn7] end connection 127.0.0.1:58562 (2 connections now open)
2017-06-21T15:17:43.621+0000 I NETWORK  [initandlisten] connection accepted from 190.7.156.123:21600 #8 (3 connections now open)

> offcorss-assistant@1.0.0 start /home/ubuntu/offcorss-assistant
> node index.js

App listening on port 80.
2017-06-21T15:17:52.578+0000 I NETWORK  [initandlisten] connection accepted from 190.7.156.123:14127 #9 (4 connections now open)
2017-06-21T15:42:10.531+0000 I NETWORK  [initandlisten] connection accepted from 190.7.156.123:54547 #10 (5 connections now open)
2017-06-21T15:42:10.841+0000 I NETWORK  [initandlisten] connection accepted from 190.7.156.123:13528 #11 (6 connections now open)
